libcontainer/process_linux.go

```diff
@@ -271,6 +294,27 @@ func (p *initProcess) createNetworkInterfaces() error {
 	return nil
 }
 
+func runCmd(command configs.Command, statePath string) error {
+	cmd := exec.Command(command.Path, command.Args[:]...)
```

Comment:
[`exec.Command`][0] doesn't let you set `argv[0]` independently of `command.Path`, which seemed like something [desired by the spec][1] (but [the spec isn't clear on how it should be handled][2]).

[0]: https://golang.org/pkg/os/exec/#Command
[1]: https://github.com/opencontainers/specs/pull/34#issuecomment-126115600
[2]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/luaaorsya10